server/sonar-server/src/main/java/org/sonar/server/measure/index/ProjectMeasuresQuery.java

```diff
@@ -31,10 +32,36 @@
 import static java.util.Objects.requireNonNull;
 
 public class ProjectMeasuresQuery {
+
+  public static final String SORT_BY_NAME = "name";
+
+  public enum Operator {
```

Comment:
I think worsen the readability. Reference: http://stackoverflow.com/a/4730156/5052331